examples/opensora_hpcai/opensora/utils/model_utils.py

```diff
@@ -4,9 +4,20 @@
 from mindspore.nn import GELU, GroupNorm, SiLU
 
 from ..models.layers.blocks import Attention, LayerNorm, LlamaRMSNorm, PositionEmbedding2D, SinusoidalEmbedding
+from ..models.text_encoder.flan_t5_large.t5 import T5LayerNorm
 
 # SORA's whitelist (FP32) operators
-WHITELIST_OPS = [LayerNorm, Attention, LlamaRMSNorm, SiLU, GELU, GroupNorm, PositionEmbedding2D, SinusoidalEmbedding]
+WHITELIST_OPS = [
+    LayerNorm,
+    Attention,
+    LlamaRMSNorm,
+    SiLU,
+    GELU,
+    GroupNorm,
+    PositionEmbedding2D,
+    SinusoidalEmbedding,
+    T5LayerNorm,
```

Comment:
`WHITELIST_OPS` are not applied to T5 embedder in `inference.py`.

Reply:
added